charts/kube-prometheus-stack/values.yaml

```diff
@@ -1748,8 +1748,8 @@ kubeScheduler:
     ## If null or unset, the value is determined dynamically based on target Kubernetes version due to change
     ## of default port in Kubernetes 1.23.
     ##
-    port: null
-    targetPort: null
+    # port: null
```

Comment:
By comment such keys out, they also omit from values.schema.json. They should be included into schema and the type should anyOf `null|number`.